$NetBSD: patch-libbio_Makefile,v 1.1 2015/04/25 19:58:32 dholland Exp $

Configure for pkgsrc; honor CC and AR.
Remove some wrong dangling references.

--- libbio/Makefile.orig	2000-02-11 17:04:17.000000000 +0000
+++ libbio/Makefile
@@ -17,19 +17,22 @@ OFILES=\
 	bseek.o\
 	bwrite.o\
 
-HFILES=/sys/include/bio.h
+#HFILES=/sys/include/bio.h
 
 INCLUDES=-I../include
 
 CFLAGS += $(INCLUDES)
 
-CC=gcc
+CC?=gcc
+
+all: $(LIB)
 
 $(LIB): $(OFILES)
-	ar r $(LIB) $(OFILES)
+	rm -f $(LIB)
+	$(AR) cq $(LIB) $(OFILES)
 
 clean:
-	rm -rf $(TARG) $(OFILES)
+	rm -rf $(OFILES)
 
 world:;		make depend; make $(LIB)
 
